packages/flutter_tools/test/general.shard/base/terminal_test.dart

```diff
@@ -42,7 +41,7 @@ void main() {
 
     setUp(() {
       terminal = AnsiTerminal(
-        stdio: globals.stdio, // Danger, using real stdio.
+        stdio: Stdio(), // Danger, using real stdio.
```

Comment:
doesn't have to be here, but from looking at this code it looks like we could replace this with a mock.